package-lock.json

```diff
@@ -53,7 +53,7 @@
         "short-unique-id": "^4.4.4",
         "styled-components": "^5.3.6",
         "swagger-ui-react": "^4.14.3",
-        "vscode": "npm:@codingame/monaco-vscode-api@~1.69.10",
+        "vscode": "npm:@codingame/monaco-vscode-api@~1.69.12",
```

Comment:
The package-lock.is is not generated properly. I suggest to revert to original versions of `vscode` dep in `package.json` and `package-lock.json`.